### DISCUSSION TITLE
Strimzi cluster authencation issue

### DISCUSSION THREAD

**Discussion by alokk-noida-techm** [strimzi/strimzi-kafka-operator#7679] [General]
I have a stimzi kafka cluster which is failing with below error, but we have the valid certificates

  WARN [Producer clientId=console-producer] Connection to node -1 (kafka1.*/*:443) terminated during authentication. This may happen due to any of the following reasons: (1) Authentication failed due to invalid credentials with brokers older than 1.0.0, (2) Firewall blocking Kafka TLS traffic (eg it may only allow HTTPS traffic), (3) Transient network issue. (org.apache.kafka.clients.NetworkClient)

Even tried to openssl command which is not showing up the acceptable client ca cert.

  openssl s_client -connect kafka1.*:443
  CONNECTED(00000005)
  write:errno=0
  ---
  no peer certificate available
  ---
  No client certificate CA names sent
  ---
  SSL handshake has read 0 bytes and written 328 bytes
  Verification: OK
  ---
  New, (NONE), Cipher is (NONE)
  Secure Renegotiation IS NOT supported
  Compression: NONE
  Expansion: NONE
  No ALPN negotiated
  Early data was not sent
  Verify return code: 0 (ok)
  ---

**Comment by scholzj** [comment]
You did not share any configurations or anything. So it is really hard to help. An error like this normally suggests you are using client with TLS enabled to connect to non-TLS listener or the other way around.
